File: 0674-longest-continuous-increasing-subsequence/0674-longest-continuous-increasing-subsequence.py
class Solution:
    def findLengthOfLCIS(self, nums: List[int]) -> int:
        if not nums:
            return
        result = 1
        maxcontinous = 1
        for num in range(1,len(nums)):
            if nums[num] > nums[num-1]:
                result += 1
            else:
                result = 1
            maxcontinous = max(maxcontinous, result)
        return maxcontinous

File: 0674-longest-continuous-increasing-subsequence/0674-longest-continuous-increasing-subsequence.java
class Solution {
    public int findLengthOfLCIS(int[] nums) {
        int n = nums.length;
        int count = 1, ans = 1;
        for (int i = 1; i < n; i++) {
            if (nums[i] > nums[i - 1])
                count++;
            else
                count = 1;
            ans = Math.max(ans, count);
        }
        return ans;
    }
}